### DISCUSSION TITLE
Show date even when taskbar icons are small

### DISCUSSION THREAD

**Discussion by papadi** [valinet/ExplorerPatcher#728] [Ideas]
Wouldn't that be great to have!? Even better if we could customize the format (e.g. I don't need to see the year, that I always know).
I know there are other tools that do that but I just don't want to install one just for this purpose.

**Comment by ghost** [comment]
I would also like this feature. I use the taskbar with small icons to gain some pixels of vertical space, but I miss this one feature of the taskbar with large icons. With large icons it shows the date, but with small icons it doesn't.

One tricky question would be how much horizontal space to leave out for the date. I would be satisfied with the space given manually in pixels as a simple solution. This could be improved later.
